lib/recurly/risk/three-d-secure/strategy/stripe.js

```diff
@@ -39,18 +39,20 @@ export default class StripeStrategy extends ThreeDSecureStrategy {
     super.attach(element);
 
     this.whenReady(() => {
-      this.stripe.handleCardAction(this.stripeClientSecret).then(result => {
+      const handleAction = this.stripeClientSecret.startsWith('pi') ? this.stripe.handleCardAction : this.stripe.confirmCardSetup;
```

Comment:
`startsWith` will not work with IE11, and won't be transpiled currently. You can use instead:

```js
this.stripeClientSecret.indexOf('pi') === 0
```